### DISCUSSION TITLE
Template literal / string interpolation doesn't work.

### DISCUSSION THREAD

**Discussion by darr1s** [dotansimha/graphql-code-generator#9600] [Q&A]
![image](https://github.com/dotansimha/graphql-code-generator/assets/21124691/d45aea9a-ec91-4963-a599-f9cc3c8c1a6b)

Hello, I have a question. I use a few reusable const for subfields (i.e. MediaField) because the backend doesn't interface it nicely where I can define just one fragment.

This doesn't seem to be recognize by the codegen. I'm thinking the codegen doesn't do typescript compile/parse and only read text as it is.

**Comment by PaulRBerg** [comment]
Related: https://github.com/dotansimha/graphql-code-generator/discussions/9495
